files/jquery.tooltipster/info.ini

```diff
@@ -1,5 +1,5 @@
 author = "Caleb Jacob"
 github = "https://github.com/iamceege/tooltipster/"
-homepage = "http://calebjacob.com/tooltipster/"
-description = "A powerful, flexible jQuery plugin enabling you to easily create semantic, modern tooltips enhanced with the power of CSS."
-mainfile = "js/jquery.tooltipster.min.js"
\ No newline at end of file
+homepage = "http://iamceege.github.io/tooltipster/"
+description = "A flexible and extensible jQuery plugin for modern tooltips."
+mainfile = "dist/js/tooltipster.bundle.min.js"
```

Comment:
Remove `dist/` here and it should be good to go.

P.S. Comment to notify me of your changes.